kod/object/active/holder/nomoveon/battler/player/user.kod

```diff
@@ -3749,6 +3758,51 @@ messages:
       return;
    }
 
+    UserMoveInventoryItem(what=$,where=$)
+   "User moved an item in their inventory. Assuming everything checks out, "
+   "we move it in the plPassive list also."
+   {
+      local iWhere, lSafeList;
+
+      if what = $
+         OR where = $
+         OR what = where
+         OR plPassive = $
+      {
+         return;
+      }
+
+      % Get list position for the drop location.
+      iWhere = FindListElem(plPassive,where);
```

Comment:
There needs to be a check here in case "where" isn't found in plPassive.

Reply:
Added checks for both payload and target objects, along with debug messages.

The debug messages fire and the function UserMoveInventoryItem returns without moving the items if either payload of target objects are not found in the plPassive list.